src/smart-components/request/request-detail/action-transcript.js

```diff
@@ -34,28 +35,37 @@ const operationInfo = {
 const operationIcon = (operation) => operationInfo[operation] ? operationInfo[operation].icon : '';
 const operationDisplayName = (operation) => operationInfo[operation] ? operationInfo[operation].displayName : untranslatedMessage();
 
-export const ActionTranscript = actionList => {
+export const ActionTranscript = ({ actionList }) => {
   const intl = useIntl();
-  const actions = actionList.actionList;
-  return actions ? (
+
+  return actionList ? (
     <Stack>
-      { actions.map(actionItem =>
+      { actionList.map(actionItem =>
         <div key={ `${actionItem.id}-action` }>
-          <TextContent><Text key={ `${actionItem.id}-action-created_at` }
-            className="pf-u-mb-0" component={ TextVariants.small }>
-            { timeAgo(actionItem.created_at) }
-          </Text>
-          <Text key={ `${actionItem.id}-action-operation` }
-            className="pf-u-mb-md">
-            { operationIcon(actionItem.operation) }&nbsp;
-            { intl.formatMessage(operationDisplayName(actionItem.operation), { by: actionItem.processed_by }) }
-          </Text>
-          { actionItem.comments && <Text key={ `${actionItem.id}-action-comments` }
-            className="pf-u-pt-0" component={ TextVariants.p }>
-            { `${actionItem.comments}` }
-          </Text> } </TextContent>
+          <TextContent>
+            <Text key={ `${actionItem.id}-action-created_at` }
+              className="pf-u-mb-0" component={ TextVariants.small }>
+              { timeAgo(actionItem.created_at) }
+            </Text>
+            <Text key={ `${actionItem.id}-action-operation` }
+              className="pf-u-mb-md">
+              { operationIcon(actionItem.operation) }&nbsp;
```

Comment:
This icon could be part of the message. That would allow us to remove the space symbol.

Reply:
This is not part of the PR. Just changed formatting to be cleaner. 😄